package.json

```diff
@@ -31,14 +31,15 @@
   "dependencies": {
     "can-assign": "^1.1.1",
     "can-event-queue": "^1.0.0",
+    "can-key": "<2.0.0",
     "can-namespace": "1.0.0",
     "can-observation": "^4.0.0",
     "can-observation-recorder": "^1.0.0",
     "can-reflect": "^1.6.1",
     "can-simple-observable": "^2.0.0",
+    "can-single-reference": "<2.0.0",
```

Comment:
I’d write `^1.0.0` since the 1.0 has already been published.

Reply:
done!